client/components/Layout/__tests__/SettingsPage.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { mount } from '@vue/test-utils';
-import SettingsPage from '~/components/Layout/SettingsPage.vue';
+import SettingsPage from '../SettingsPage.vue';
```

Comment:
```suggestion
import SettingsPage from '~/components/Layout/SettingsPage.vue';
```
You can put that back, tested and approved!

Reply:
I think he did the change not because it didn't work, but because it's more "correct". 

The tests should always be alongside the components that are being tested, so if at some point in the future we move the component, the test should be alongside it and the import will work regardless.

Reply:
> I think he did the change not because it didn't work, but because it's more "correct".

That's in part true :yum: It didn't work, so it was half an attempt to make it work, and also to make it easier to move tests around in the future.

However, I reverted them all back for now, since I think it's better to do that in a followup PR, if we want to do it.